packages/models-library/src/models_library/api_schemas_webserver/wallets.py

```diff
@@ -21,6 +21,10 @@ class WalletGet(OutputSchema):
     created: datetime
     modified: datetime
 
+    model_config = ConfigDict(
+        frozen=False
```

Comment:
Superclass is `frozen` and the behaviour was inherited.

Reply:
same question

Reply:
In a test there was a direct assignment of a field in that model.